micromamba/tests/test_create.py

```diff
@@ -652,6 +652,23 @@ def test_spec_with_channel_and_subdir():
         )
 
 
+def test_spec_with_slash_in_channel(tmp_home, tmp_root_prefix):
+    env_name = "myenv"
+    try:
+        res = helpers.create("-n", env_name, "pkgs/main/noarch::python", "--dry-run")
+    except subprocess.CalledProcessError as e:
+        assert e.stderr.decode() == (
+            'critical libmamba The package "pkgs/main/noarch::python" is '
+            "not available for the specified platform\n"
+        )
+
+    try:
+        os.environ["CONDA_SUBDIR"] = "linux-64"
+        helpers.create("-n", env_name, "pkgs/main/linux-64::python", "--dry-run")
+    finally:
+        os.environ.pop("CONDA_SUBDIR")
```

Comment:
Can we also try with `"pkgs/main::python"` to make sure that "unknown" subdirs are indeed allowed now? Thanks! 🙏